//package ExceiptionHandling;
//
//
//
//public class ExceptionUsingThrow {
//    public static void main(String[] args) {
//        throw new ArrayIndexOutOfBoundsException("faltu index");
//        System.out.println("Hello");
//    }
//}
